packages/plugins/Validation/lib/locales/en.js

```diff
@@ -17,7 +17,9 @@ export default {
         between: '{_field_} should be between {prev} and {next}'
       },
       number: '{_field_} must be valid number',
-      sameAsPassword: 'Passwords must be identical'
+      equal: '{_field_} must be identical',
```

Comment:
Without name of another field this message is useless.  Field must be identical to what?